main_server.py

```
@@ -1362,19 +1362,25 @@ def _is_port_available(port: int) -> bool:
     print(f"启动配置: {get_start_config()}")
 
     # 2) 信号处理：Ctrl+C 时快速关闭
+    #    uvicorn 的 install_signal_handlers() 会用 signal.signal(sig, self.handle_exit)
+    #    覆盖我们直接注册的信号处理器。所以这里 monkey-patch server.handle_exit，
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)